### PR TITLE
set `engines` version to `>=14`

### PR DESCRIPTION
**Pull request by KATT**
node.16 is LTS now

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "check-changed-files": "yarn ts-node scripts/ts-check-changed-files.ts"
   },
   "engines": {
-    "node": "14.x",
+    "node": "14.x || 16.x",
```
Comment (KATT) [inline package.json:29]:
```suggestion
    "node": ">=14.x",
```

**Review by KATT — COMMENTED** [review]
this is prob btter

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "check-changed-files": "yarn ts-node scripts/ts-check-changed-files.ts"
   },
   "engines": {
-    "node": "14.x",
+    "node": ">=14.x",
```
Comment (mihaic195) [inline package.json:29]: There may be a problem using 16 with Railway deployments. I don't remember exactly, maybe @emrysal knows.

Comment (emrysal) [reply package.json:29]: There is, but on the other hand we should be looking at the Railway deployment itself; why is it failing; in addition there is many other reasons it fails at the moment due to outdated env vars, needs updating.

**Review by emrysal — APPROVED** [review]

**Pull request merged**